src/Models/DataImport.php

```diff
@@ -210,4 +212,16 @@ public function import(?string $sheet = null)
 
         Import::dispatch($this, $sheet);
     }
+
+    public function restart(): self
+    {
+        $this->when(
+            $this->template()->type() === ImportType::Insert,
+            fn () => $this->fill(['failed' => -1 * $this->successful]),
```

Comment:
this will produce a lot of awkward situations. Imagine failed imports with -233 failed.

I could live with resetting both to 0 on all cases